### PR TITLE
File selection

### PR DESCRIPTION
**Pull request by MStraatman**
Added a file selection option in the application

### CODE REVIEW

File: src/main/java/Window/Controller.java [hunk 1]
```diff
@@ -1,22 +1,75 @@
...
+        Stage stage = (Stage) menu.getScene().getWindow();
+        Parent root;
+        if (event.getSource() == openFile) {
+            final FileChooser fileChooser = new FileChooser();
+            FileChooser.ExtensionFilter extFilter = new FileChooser.ExtensionFilter("GFA files (*.gfa)", "*.gfa");
+            fileChooser.getExtensionFilters().add(extFilter);
+            final Button openButton = new Button("Open");
+            File file = fileChooser.showOpenDialog(stage);
+            if (file != null) {
+                //TODO open new file
```
Comment (jjtiessen) [inline src/main/java/Window/Controller.java]: What should be done here?

File: src/main/java/Window/App.java [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
+package window;
 
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
+import javafx.scene.text.Text;
+import javafx.scene.layout.GridPane;
 import javafx.stage.Stage;
 
+
```
Comment (jjtiessen) [inline src/main/java/Window/App.java]: Unnecessary empty line.

File: src/main/java/Window/Controller.java [hunk 3]
```diff
@@ -1,22 +1,75 @@
...
 public class Controller {
+    @FXML private Button browse;
+
+    @FXML private MenuBar menu;
+
+    @FXML private MenuItem openFile;
+
     @FXML private TextArea console;
 
+    @FXML private void open(final ActionEvent event) throws IOException {
```
Comment (jjtiessen) [inline src/main/java/Window/Controller.java]: I seems like this method could handle multiple types of opening request. I suggest you specialize this method to only be invoked when openFile is clicked.

Comment (MStraatman) [inline src/main/java/Window/Controller.java]: Valid point

File: src/main/java/Window/Controller.java (hunk 1 above)
Comment (MStraatman) [inline src/main/java/Window/Controller.java]: Here the file should be parsed and visualized, but that's not working yet, so not included here. Will update the TODO to make that clearer.

File: src/main/java/Window/App.java (hunk 2 above)
Comment (MStraatman) [inline src/main/java/Window/App.java]: Removed it

File: src/main/java/Window/Controller.java [hunk 4]
```diff
@@ -33,15 +33,13 @@
     @FXML private void open(final ActionEvent event) throws IOException {
```
Comment (jjtiessen) [inline src/main/java/Window/Controller.java]: Method name is still too general. I suggest changing it to something like "openNewFile".

Comment (MStraatman) [inline src/main/java/Window/Controller.java]: Sure

**Pull request merged**